roles/infrastructure/tasks/setup_azure_storage.yml

```diff
@@ -90,6 +109,13 @@
         service_level: "{{ infra__azure_netapp_vol_type }}"
         size: "{{ infra__azure_netapp_vol_size }}"
 
-    - name: Prepare netapp vol info for submission
+    - name: Prepare netapp vol info for submission during initial creation
+      when: __azure_netapp_vol_details.msg is defined
+      set_fact:
+        infra__azure_nfs_mount: "{{ __azure_netapp_vol_details.msg }}"
+
+    # Yay for Azure consistency
+    - name: Prepare netapp vol info for submission during recreation
```

Comment:
This can take a long time (3-6 minutes). It's MORE than a fair assumption to say that it will be done by the time the CDP environment is up. Just wondering out loud if there's any benefit to waiting for it.

Reply:
Because it fails in weird ways, especially due to backend Netapp actions that sometimes just fail and require an Azure support ticket to resolve. Therefore the user may get a weird CML deployment failure when actually their NFS is the problem.